Resistance.py

```diff
@@ -0,0 +1,355 @@
+import os
+os.environ['XLA_PYTHON_CLIENT_PREALLOCATE'] = 'false'
+# import time
+from functools import partial
+from jax import jit
+import jax.numpy as jnp
+from jax_md import space
+from jax.config import config
+config.update("jax_enable_x64", True)
+
+ 
+@partial(jit, static_argnums=[2,3])
+def RFU_Precondition(
+        ichol_relaxer,
+        R,
+        N,
+        n_pairs_lub_prec,
+        nl_lub_prec,
+        ):
+    
+        # Load resistance table
+        ResTable_dist = jnp.load('ResTableDist.npy')
+        ResTable_vals = jnp.load('ResTableVals.npy')
+        ResTable_min = 0.0001000000000000
```

Comment:
Is there a reason to add the zeros at the end?

Reply:
No. It is now just ResTable_min = 0.0001. Same for below.